dubbo-remoting/dubbo-remoting-netty4/src/main/java/org/apache/dubbo/remoting/transport/netty4/NettyTransporter.java

```diff
@@ -32,7 +33,8 @@ public class NettyTransporter implements Transporter {
 
     @Override
     public RemotingServer bind(URL url, ChannelHandler handler) throws RemotingException {
-        return new NettyServer(url, handler);
+//        return new NettyServer(url, handler);
+        return new NettyPortUnificationServer(url, handler);
```

Comment:
Do not replace the original logic. `NettyPortUnificationServer` should be created with `NettyPortUnification Transporter`